vault/seal/ocikms/ocikms.go

```diff
@@ -0,0 +1,414 @@
+// Copyright © 2019, Oracle and/or its affiliates.
+package ocikms
+
+import (
```

Comment:
Can you group standard library at the top and third party libraries split by a newline and in a second group?